src/components/follow-in-lower-fov.js

```diff
@@ -2,7 +2,8 @@ AFRAME.registerComponent("follow-in-lower-fov", {
   schema: {
     target: { type: "selector" },
     offset: { type: "vec3" },
-    speed: { type: "number", default: 0.003 }
+    speed: { type: "number", default: 0.003 },
+    angle: { type: "number", default: Math.PI / 4 }
```

Comment:
Would be nice to allow specifying this in degrees and convert to radians in `update`